### PR TITLE
Differentiate introspection response and access tokens & register media type

### PR DESCRIPTION
**Pull request by tlodderstedt**
intended to resolve issues #1 and #4

### CODE REVIEW

**Review by vdzhuvinov — COMMENTED** [review]
Reword line 225 to "This JWT is used to cryptographically protect the token introspection response and is not intended to be used as an access token itself"?

**Comment by tlodderstedt** [comment]
The point I want to make is: this is not the access token passed to the Introspection Endpoint (or a representation of it).

**Comment by vdzhuvinov** [comment]
Alright. To add extra clarity, because people having opaque tokens might also wonder how the token is a JWT, how about

"For a JWT-encoded access token the introspection response is not the access token itself. Implementers MUST consider this difference..."

**Comment by tlodderstedt** [comment]
changed text - please give it another read

File: draft-ietf-oauth-jwt-introspection-response.xml [hunk 1]
```diff
@@ -124,17 +124,16 @@
...
-      assurance that the authorization server issued the access token,
-      including cases where the authorization server assumes liability for the
-      token's content. An example is a resource server using verified person
-      data to create certificates, which in turn are used to create qualified 
-      electronic signatures.</t>
+      assurance that the authorization server issued the token introspection 
+      response for an access token, including cases where the authorization server 
+      assumes liability for the content of the token introspection response. 
+      An example is a resource server using verified person data to create certificates, 
+      which in turn are used to create qualified electronic signatures.</t>
```
Comment (tlodderstedt) [inline draft-ietf-oauth-jwt-introspection-response.xml:136]: Test

**Comment by tlodderstedt** [comment]
@vdzhuvinov can you please review the PR again?

**Comment by vdzhuvinov** [comment]
Much improved text, great!

**Pull request merged**